From 1b9762932254e57ebdbc6cc5aaf3e6313ccbb27e Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 12 Oct 2023 18:07:44 +0200
Subject: [PATCH] arm64: dts: st: add power domain in mdf node on stm32mp251

Add power domain for MDF on STM32MP251 device tree.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I4a350181a236e4b32bd9583dbc43ea1bd82b6d71
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 0c5ec9ca3c81..56612880d8ed 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1710,6 +1710,7 @@ mdf1: mdf@404d0000 {
 				resets = <&rcc MDF1_R>;
 				reset-names = "mdf";
 				access-controllers = <&rifsc 54>;
+				power-domains = <&RET_PD>;
 				status = "disabled";
 
 				sitf0: sitf@80 {
-- 
2.39.5

